tests/validators/tests.py

```diff
@@ -325,19 +324,209 @@
     (ProhibitNullCharactersValidator(), None, None),
 ]
 
+VALID_URLS = [
```

Comment:
I would put both of these before `TEST_DATA` because they are simpler.

Reply:
Sure.